apps/web/src/pages/PasswordResetPage/PasswordResetPage.tsx

```diff
@@ -39,7 +78,7 @@ export default function PasswordResetPage(): JSX.Element {
         </div>
 
         {/* Negative Alert Banner */}
-        {displayError.length > 0 && (
```

Comment:
Typically in JS, checking for a string by looking at it's length is a bit odd because an empty string evaluates to false in JS and seeing .length makes me think its an array.

Reply:
I don't think this is an issue, it seems to be getting the job done.